domain/src/main/java/tachiyomi/domain/release/interactor/GetApplicationRelease.kt

```diff
@@ -81,6 +83,8 @@ class GetApplicationRelease(
         val commitCount: Int,
         val versionName: String,
         val repository: String,
+        val changeRepository: (String) -> Unit = {},
+        val kujakuKey: String = "",
```

Comment:
Do these need a default, is GetApplicationRelease used anywhere else?

Reply:
in test